src/components/LearningMaterial/index.tsx

```diff
@@ -38,10 +41,16 @@ export default function LearningMaterial({
   publishedAt,
   updatedAt,
   locale,
+  courseCopy,
 }: Props) {
   const { locale: routerLocale } = useRouter()
-  const updatedText = updatedAt ? `Updated at ${formatDate(updatedAt)} ` : ''
-  const createdText = publishedAt ? `Created at ${formatDate(publishedAt)}` : ''
+
+  const updatedText = updatedAt
+    ? `${courseCopy?.CourseUpdatedAt} ${formatDate(updatedAt)} `
+    : ''
```

Comment:
As you know, this component will be used from all different landning pages, meaning it's a bit unclean to have a prop that is specifically tied to 'course'.

Maybe we can pass in the whole updatedText as a prop instead? And keep the course/lecture/block-specific logic about its creation on each landning page instead.

Reply:
Good point. Thanks